tests/sentry/web/frontend/test_auth_login.py

```diff
@@ -252,12 +252,22 @@ def test_redirects_already_authed_non_superuser(self):
             resp = self.client.get(self.path)
             self.assertRedirects(resp, "/organizations/new/")
 
-    def test_doesnt_redirect_already_authed_superuser(self):
```

Comment:
The behaviour of the view has changed as superuser access isn't granted upon login, so we no longer need to redirect a non active superuser